jbpm-services/jbpm-kie-services/pom.xml

```diff
@@ -11,6 +11,10 @@
   <packaging>bundle</packaging>
   <name>jBPM :: KIE Services</name>
 
+  <properties>
+    <osgi.Bundle-SymbolicName>org.jbpm.kie.services.api.jpa;singleton:=true</osgi.Bundle-SymbolicName>
```

Comment:
jpa should not be in the bundle name - it was a typo previously. So I think it would make sense to fix it no. I don't think services api is used anywhere in osgi